src/style.css

```diff
@@ -1 +1,58 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 300;
+}
+
+body {
+  padding: 0;
+}
+
+.search {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: Avenir, sans-serif;
```

Comment:
Ensure that your CSS includes a fallback font for `font-family: Avenir;` to maintain proper rendering if the primary font is unavailable. You have correctly added `sans-serif` as a fallback.